PyDSS/metrics.py

```diff
@@ -454,16 +486,21 @@ def _update_counts(self, reg_control):
         last_value = self._last_values[reg_control.FullName]
         if last_value is not None:
             self._change_counts[reg_control.FullName] += abs(tap_number - last_value)
-            logger.debug("%s changed count from %s to %s count=%s",
-                         reg_control.Name, last_value, tap_number,
-                         self._change_counts[reg_control.FullName])
+            logger.debug(
+                "%s changed count from %s to %s count=%s",
+                reg_control.Name,
+                last_value,
+                tap_number,
+                self._change_counts[reg_control.FullName],
+            )
 
         self._last_values[reg_control.FullName] = tap_number
 
 
 class OpenDssExportMetric(MetricBase):
     def __init__(self, prop, dss_objs, options):
         super().__init__(prop, dss_objs, options)
+        self._tmp_dir = tempfile.mkdtemp()
```

Comment:
The old code was causing OpenDSS to write the exported files to Lustre, which resulted in max times of 4 seconds. This change reduces the max to under a second.